newsroom/cards/views.py

```diff
@@ -28,6 +29,10 @@ def get_settings_data():
     }
 
 
+def _get_file_from_request(key: str) -> Optional[Any]:
```

Comment:
This might be useful in the new [newsroom.flask](https://github.com/superdesk/newsroom-core/blob/async/newsroom/flask.py) module, so we can support flask features we use from the one file (until we add this feature into the new Endpoint Request/Response system)